go/staking/api/address.go

```diff
@@ -0,0 +1,64 @@
+package api
+
+import (
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/address"
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/signature"
+	"github.com/oasisprotocol/oasis-core/go/common/encoding/bech32"
+)
+
+// Human readable part of Bech32 encoded addresses.
+const Bech32HRP = "osa"
```

Comment:
Do we also want to ensure the Bech32 HRPs (human readable parts) are unique (like versions and contexts)?

Note that I temporarily used `osa` for _oasis staking address_, but we need to decide together on what to use.

Reply:
Yeah that would be useful. Also the above constant should be renamed to something like `AddressBech32HRP` as otherwise it is not clear what it is for.

Reply:
Ok, I'll add HRP registration and rename the constant.

Reply:
Done.